ScriptAnalyzer2/Builtin/BuiltinRuleProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.PowerShell.ScriptAnalyzer.Builder;
+using Microsoft.PowerShell.ScriptAnalyzer.Builtin.Rules;
+using Microsoft.PowerShell.ScriptAnalyzer.Configuration;
+using Microsoft.PowerShell.ScriptAnalyzer.Execution;
+using Microsoft.PowerShell.ScriptAnalyzer.Runtime;
+using Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Runspaces;
+
+namespace Microsoft.PowerShell.ScriptAnalyzer.Builtin
+{
+    public static class BuiltinRules
+    {
+        public static IReadOnlyList<Type> DefaultRules { get; } = new[]
+        {
+            typeof(AvoidEmptyCatchBlock),
+            typeof(AvoidGlobalVars),
+            typeof(AvoidPositionalParameters),
+            typeof(AvoidUsingWMICmdlet),
+            typeof(UseDeclaredVarsMoreThanAssignments),
+            typeof(UseShouldProcessForStateChangingFunctions),
+        };
+    }
+
+    public static class Default
+    {
+        private static readonly Lazy<IRuleComponentProvider> s_ruleComponentProviderLazy = new Lazy<IRuleComponentProvider>(BuildRuleComponentProvider);
+
+        public static IReadOnlyDictionary<string, IRuleConfiguration> RuleConfiguration { get; } = new Dictionary<string, IRuleConfiguration>(StringComparer.OrdinalIgnoreCase)
+        {
+            { "PS/AvoidUsingEmptyCatchBlock", null },
```

Comment:
I think some of this will change. The key part though is that I want to improve cold start performance by eliminating reflection and should also have a nice declarative way to list rules.